packages/compass-web/src/entrypoint.tsx

```diff
@@ -280,6 +278,7 @@ const CompassWeb = ({
     onDebug,
   });
   const preferencesAccess = useCompassWebPreferences(initialPreferences);
+  // TODO: My Queries feature flag will be used to conditionally provide storage providers in COMPASS-9565
```

Comment:
```suggestion
  // TODO(COMPASS-9565): My Queries feature flag will be used to conditionally provide storage providers
```

(just a format we've tried to standardize on in the past)